kyo-core/jvm/src/main/scala/kyo/internal/LogPlatformSpecific.scala

```diff
@@ -17,42 +17,42 @@ object LogPlatformSpecific:
             def apply(name: String) = new SLF4J(org.slf4j.LoggerFactory.getLogger(name))
 
         class SLF4J(logger: org.slf4j.Logger) extends Log.Unsafe:
-            def level =
```

Comment:
The main bug was here. The logic was inverted.

Reply:
I think it'd be an issue only if the SLF4J config dynamically changes at runtime. I've never used such a feature, I'm not sure if it's possible or how useful it'd be. I made it a `val` because, in theory, we'd need an `IO` suspension to read the level.